experimental flag polyfillsOptimization  does not work with this module

Hi Guys

I tried to enable 
```
experimental: {
      modern: true,
      polyfillsOptimization: true
}
```
This results into following error: 
```
ModuleNotFoundError: Module not found: Error: Can't resolve '@babel/runtime-corejs2/core-js/array/is-array'
```
This is early adopter code so I just wanted to bring this up in case the feature finally lands into NextJS core

I don't believe this is an experimental issue.  I just updated to next.js 9.2.3-canary.8 to test something and (without adding any experimental flags) got:
```
[ error ] ./node_modules/next-seo/lib/jsonld/localBusiness.js
Module not found: Can't resolve '@babel/runtime-corejs2/core-js/array/is-array' in '\node_modules\next-seo\lib\jsonld'
```

--- a/.babelrc
+++ b/.babelrc
@@ -1,5 +1,9 @@
 {
-  presets: ['next/babel', '@babel/typescript', 'minify'],
+  presets: [
+    ['next/babel', { 'transform-runtime': { corejs: false } }],
+    '@babel/typescript',
+    'minify',
+  ],
   plugins: ['@babel/plugin-transform-react-jsx'],
   ignore: ['node_modules'],
 }

